src/tmate.py

```diff
@@ -120,11 +163,11 @@ def start_daemon(address: str) -> None:
     try:
         systemd.daemon_reload()
     except systemd.SystemdError as exc:
-        raise DaemonStartError("Failed to reload tmate-ssh-server daemon") from exc
+        raise DaemonStartError("Failed to reload tmate-ssh-server daemon.") from exc
     try:
-        systemd.service_start("tmate-ssh-server")
+        systemd.service_start(TMATE_SERVICE_NAME)
```

Comment:
You can catch both exceptions together, as they are the same. The trace will point to the specific failed instruction so I don't see the need for different error messages here. The specifics of the service failing to start are irrelevant to the caller

Reply:
Makes sense, updated :)